configure

```diff
@@ -345,7 +345,7 @@ fi
 # only update dependencies, when we are not in a release tarball
 if [ -d .git  -a $SKIP_DEPS -ne 1 ]; then
     echo "==> updating dependencies"
-    ${REBAR} get-deps update-deps
+    ${REBAR3} get-deps update-deps
```

Comment:
There is no `update-deps` command. Also, you can't run multiple commands this way but have to use `do`:

```
rebar3 do compile, ct
```

You also shouldn't need to run `get-deps` ever. It is a backwards compat command we added after the fact even though it isn't really needed -- though can be useful like when building a docker image to first fetch the deps into a layer. So you may just be wanting `rebar3 upgrade` here.